### PR TITLE
Stathat

### PR DESCRIPTION
**Pull request by angaither**
Adds stathat module
send donation stats to stathat

### CODE REVIEW

File: lib/modules/dosomething/dosomething_payment/dosomething_payment.module [hunk 1]
```diff
@@ -210,6 +210,10 @@ function dosomething_payment_form_submit($form, &$form_state) {
   $values['amount'] = $values['amount'] * 100;
   // Send Payment request to Stripe API.
   if (dosomething_payment_post_payment_stripe($values)) {
+    // Send stathat info.
+    if (module_exists('stathat')) {
+      stathat_send_ez_count('drupal - Donate', 1);
```
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_payment/dosomething_payment.module:215]: Minor, but I'm not sure if the module handles when the ez key hasn't been entered. Is the plan to manually enter the key on stage and prod?

**Comment by aaronschachter** [comment]
Code looks good, just need to check for minor edge case where form may be submitted before key is entered (or if erased)..

File: lib/modules/dosomething/dosomething_payment/dosomething_payment.module (hunk 1 above)
Comment (angaither) [inline lib/modules/dosomething/dosomething_payment/dosomething_payment.module:215]: yarrr, it just wouldn't send which isn't worse than what's happening now, right?

**Comment by angaither** [comment]
wouldn't really want to hardcode the key into code...

**Comment by aaronschachter** [comment]
Yah, think its fine for now since it's just Donate Page, i'm just not sure if the module throws an error/watchdog if the key doesnt exist, if it gets wiped out by chance.

**Pull request merged**